exporter/coralogixexporter/client.go

```diff
@@ -64,17 +64,20 @@ func (c *coralogixClient) startConnection(ctx context.Context, host component.Ho
 	return nil
 }
 
-func (c *coralogixClient) newPost(ctx context.Context, td pdata.Traces) {
+func (c *coralogixClient) newPost(ctx context.Context, td pdata.Traces) error {
 	batches, err := jaeger.InternalTracesToJaegerProto(td)
 	if err != nil {
 		c.logger.Error("cant translate to jaeger proto ", zap.Error(err))
```

Comment:
When returning the error, don't log it here, as it will be logged elsewhere up the call chain already.

Reply:
@jpkrohling I want the user will get an indicative message, in addition to the default error.

Reply:
I need a second opinion from another approver here as well (@open-telemetry/collector-contrib-approvers).

Reply:
If the error is going to be logged further up the call stack but you want to ensure that the logged message contains context regarding where specifically the error was thrown and in what circumstances, that is a situation where the error should be wrapped before it is returned.

Instead of this:

```go
return err
```

do this:

```go
return fmt.Errorf("can't tpanslate to jaeger proto: %w", err)
```

Reply:
Thanks, @jpkrohling and @Aneurysm9 I will fix it.